packages/terra-functional-testing/src/services/wdio-visual-regression-service/methods/BaseCompare.js

```diff
@@ -105,7 +105,7 @@ class BaseCompare {
 
     // Added to allow for test reusability from terra repositories
     if (specPath.includes('node_modules')) {
-      [, specPath] = specPath.split('node_modules');
+      specPath = specPath.replace('node_modules', path.join('tests', 'wdio'));
```

Comment:
since we are no longer using Array.split() can we change [specPath definition](https://github.com/cerner/terra-toolkit/blob/b4c3235c97562900a7d405ed3b096bbba318dd8c/packages/terra-functional-testing/src/services/wdio-visual-regression-service/methods/BaseCompare.js#L104) to 
```js
let specPath;
```

Reply:
I think we can remove `this.baseScreenshotDir` as that is  being used to add `process.cwd()`, which we are removing at the `specPath` definition using `Array.split()`

Reply:
This was doing the pretty much the same thing as in [terra-toolkit-boneyard](https://github.com/cerner/terra-toolkit-boneyard/blob/75aaf518227320750d603f27f72536873b6e6b08/config/wdio/visualRegressionConf.js#L74). I wonder where the difference is between the two.